tests/epyccel/test_arrays.py

```diff
@@ -3161,7 +3098,7 @@ def test_array_1d_slice_stride_22(language):
         pytest.param("fortran", marks = pytest.mark.fortran),
         pytest.param("python", marks = pytest.mark.python),
         pytest.param("c", marks = [
-            pytest.mark.xfail(reason="Negative strides in slices are not handled"),
+            pytest.mark.xfail(reason="Negative strides in slices are not handled. See #1311"),
```

Comment:
Same here.

Reply:
I am now using the same error message throughout: [13b6ba3](https://github.com/pyccel/pyccel/pull/2297/commits/13b6ba3ba68b23936c8986a81120d134c837b456)